src/osdatahub/utils.py

```diff
@@ -157,7 +157,10 @@ def address_to_feature(address, crs):
         attributes called 'LAT' and 'LNG' in the address
     """
     if crs.lower() in ("epsg:27700", "bng"):
-        x, y = address["X_COORDINATE"], address["Y_COORDINATE"]
+        try:
```

Comment:
I think it's usually better practice to check what keys you have explicitly, instead of a try/except. E.g. if "X_COORDINATE" in address: x, y = ...